common/utils.js

```diff
@@ -464,7 +464,8 @@
                 appPath = ContextUtils.getValueFromContext(appContextMapping, context);
             }
 
-            var url = chaiseBaseURL() + appPath + "/#" + location.catalog + "/" + location.path;
+            // slice to remove trailing '/' instead of modifying appPath
```

Comment:
We previously wrote in a comment that `chaiseDeploymentPath()` will return a leading and trailing `/` character. 

I could update appPath to remove the leading `/` to be consistent with this change but it seems like this is the only case where it's an issue so I modified the string specifically for this case instead.